File: python-solutions/problem001.py
from time import time

def solution():
    return sum(filter(lambda x: x%3 == 0 or x%5 == 0, range(0, 1000)))

if __name__ == "__main__":
    start  = time()
    answer = solution()
    print(f"\nAnswer to Problem 1: {answer}")
    print(  f"Time Taken: {time()-start} seconds\n")

File: golang-solutions/problem001.go
package main

import (
	"fmt"
	"time"
)

func main() {
	start := time.Now()
	sum := 0
	for i := 1; i < 1000; i++ {
		if i%3 == 0 || i%5 == 0 {
			sum += i
		}
	}

	fmt.Printf("\nAnswer to Problem 1: %d\n", sum)
	fmt.Printf("Time Taken: %v\n\n", time.Since(start))
}
